flake.nix

```diff
@@ -17,5 +30,16 @@
           ];
           shellHook = "go mod tidy";
         };
+        devShells.gon = if system == "x86_64-darwin" || system == "aarch64-darwin" then
+          pkgs.mkShell {
+            buildInputs = with pkgs; [
+              gon
+            ];
+            shellHook = ''
+              export PATH=/usr/bin:$PATH:${gon}/bin
```

Comment:
```
export PATH=/usr/bin
```

is needed since `xcrun notarytool` isn't available from the `/nix/store` but is needed by `gon`